src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -45,6 +45,18 @@ const Icons = {
 }
 
 export class BidResult extends React.Component<BidResultProps> {
+  backButtonListener?: NativeEventSubscription = undefined
+
+  componentDidMount = () => {
+    this.backButtonListener = BackHandler.addEventListener("hardwareBackPress", this.handleBackButton.bind(this))
```

Comment:
nonblocking

```suggestion
    this.backButtonListener = BackHandler.addEventListener("hardwareBackPress", this.handleBackButton)
```

it's already bound